internal/common/config/config.go

```diff
@@ -25,17 +25,66 @@ type Config struct {
 
 	// MaxCriticalConsecutiveTurns is the maximum consecutive turns an island can be in the critical state.
 	MaxCriticalConsecutiveTurns uint
+
+	// Wrapped foraging config
+	ForagingConfig ForagingConfig
+
+	// Wrapped disaster config
+	DisasterConfig DisasterConfig
+}
+
+// ForagingConfig captures foraging-specific config
+type ForagingConfig struct {
+	// Deer Hunting
+	MaxDeerPerHunt        uint    // Maximimum possible number of deer on a single hunt (regardless of number of participants)
+	IncrementalInputDecay float64 // Determines decay of incremental input cost of hunting more deer
+	BernoulliProb         float64 // `p` param in D variable (see README). Controls prob of catching a deer or not
+	ExponentialRate       float64 // `lambda` param in W variable (see README). Controls distribution of deer sizes.
+
+	// Deer Population
+	MaxDeerPopulation     uint    // Maximimum possible deer population. Reserved for post-MVP functionality
+	DeerGrowthCoefficient float64 // Scaling parameter used in the population model. Larger coeff => deer pop. regenerates faster
```

Comment:
Something to keep in mind for later is to perhaps make the ForagingConfig less specific to deer as I imagine all of these parameters would also be wanted for things like fish (so MaxDeerPopulation => MaxPopulation) but this is perfectly fine for MVP

Reply:
+1 maybe for each type of foraging we need enums or something like that...

Reply:
I think we plan to add separate analogs for fishing or other foraging forms - like MaxFishStock or whatever. Would that make sense?